lib/veritech-server/src/config.rs

```diff
@@ -76,6 +82,15 @@ pub struct Config {
 
     #[builder(default = "random_instance_id()")]
     instance_id: String,
+
+    #[builder(default = "default_enable_pause_resume_stream()")]
+    enable_pause_resume_stream: bool,
+
+    #[builder(default = "default_pause_duration()")]
+    pause_duration: Duration,
+
+    #[builder(default = "default_reconnect_backoff_duration()")]
+    reconnect_backoff_duration: Duration,
```

Comment:
If this becomes a standard pattern, should this stuff get packed into the NatsConfig?

Reply:
I guess it's server specific, so maybe not